swift/third_party/resource-dir/BUILD.bazel

```diff
@@ -0,0 +1,27 @@
+alias(
+    name = "resource-dir",
+    actual = select({"@platforms//os:" + os: "@swift-resource-dir-" + os for os in ("linux", "macos")}),
+    target_compatible_with = select({
+        "@platforms//os:windows": ["@platforms//:incompatible"],
+        "//conditions:default": [],
+    }),
+    visibility = ["//swift:__pkg__"],
+)
+
+[
+    sh_binary(
```

Comment:
Is this not easily done in a genrule, instead of having to deal with `runfiles.bash`?

Reply:
this is a dev tool to update an in-tree artifact, so `genrule` won't help.